github_pr.py

```diff
@@ -237,8 +255,10 @@ def main():
         github_update_pr(**args)
 
     gh = Github(args['token'])
-    if not args['numberonly'] and not args['noratelimit']:
-        print "Github Rate Limiting: %d remaining of max %d" % (gh.rate_limiting[0], gh.rate_limiting[1])
+
+    if (('numberonly' in args) and not args['numberonly']):
```

Comment:
its not ideal because of the double negatives but it does work well. I tried lots of variations on setting the default values to false for both of those but couldn't figure out the logic in the end.